README.md

```diff
@@ -158,6 +158,24 @@ end
 
 Bug reports and pull requests are welcome on GitHub at [ministryofjustice/laa-fee-calculator-client](https://github.com/ministryofjustice/laa-fee-calculator-client)
 
+## Publishing
+
+1. Make required changes, run specs (rerecord vcr cassettes if necessary), code-review, merge etc.
+
+2. Run `bin/ruby_version_test` to test against ruby versions (2.4+ support at present)
+
+3. Update the version in `lib/laa/fee_calculator/version`
+
+4. Build the gem
```

Comment:
Between steps **4** and **5** we should probably have an additional one to create a tag for the new version before publishing it.

I've actually came across [this](https://github.com/alphagov/gem_publisher) by accident, which seems to provide a rake task to perform all that in one go. We don't need to use it, but I think the idea behind it is neat, as we're likely to want to enforce those steps in sequence so that we always have a tagged version of the published gem.

Reply:
will add a step in the readme but could you raise as github issue for this additional automation

Reply:
Done! ;) Here it is FYI: https://github.com/ministryofjustice/laa-fee-calculator-client/issues/17